hips/hip-9999.md

```diff
@@ -0,0 +1,80 @@
+---
+hip: 9999
+title: "Helm Triage Maintainers"
+authors: [ "Matthew Fisher <matt.fisher@microsoft.com>" ]
+created: "2021-07-06"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+The existing Helm Project Maintainers have identified the need for a new "triage" role. This document lists out the responsibilities of this new role.
+
+## Motivation
+
+Helm Project Maintainers are responsible for activities surrounding the development and release of code, the operation of any services they own (e.g. <https://get.helm.sh>), or the tasks needed to execute their project (e.g., community management, setting up an event booth). Introducing a new "Helm Triage Maintainer" role allows new project maintainers to join the project without the responsibilities of release management, service operation, or event management.
+
+This new role provides three benefits:
+
+1. New triage maintainers can help maintain the project's day-to-day, such as responding to new tickets and review pull requests
+2. Helm project maintainers' burden is reduced, allowing them to carry out higher-level tasks like coordinating software releases, organizing events, and maintaining production services.
+3. Provides an easier on-ramp from community members, to triage maintainers, to project maintainers, to org maintainers.
+
+## Rationale
+
+Project maintainers are looking for new members to join the project. However, community members have expressed in the past that they cannot commit more than a few hours per month to the project. Additionally, some project maintainers are concerned providing newer members the "keys to the castle" prior to a formal vetting process may introduce significant risk to the project's users. A new maintainer may be available to help maintain the project, but it may take several months before trust is established.
+
+## Specification
+
+The new triage maintainer role is allowed to perform certain day-to-day tasks as a maintainer of the project, including:
+
+- attach labels to issues
+- assign issues to milestones
+- receive access to the cncf-helm-core-maintainers mailing list
+- vote on governance-related topics
+- review and approve pull requests
+- merge pull requests
+
+The following roles and resposibilities are NOT allowed to be performed under this role:
+
+- release new revisions of the Helm Client
+- operate any public-facing services, such as
+   - <https://get.helm.sh>
+   - Mailing lists
+   - Slack channels
```

Comment:
Does this mean we would discourage triage maintainers from answering questions in the #helm-users channel of k8s slack?

Reply:
Nope! Really all of the channels are administered by the CNCF. This is just saying triage maintainers should not be the owners of official communication channels.